src/Dfc.CourseDirectory.WebV2/Features/TLevels/DeleteTLevel/View.cshtml

```diff
@@ -57,7 +57,7 @@
                     </govuk-checkboxes-fieldset-legend>
                 </govuk-checkboxes-fieldset>
                 <govuk-checkboxes-item value="@(true)">
-                    Yes - I want to delete the T level
+                    Yes - I want to delete this T level
```

Comment:
Wasn't there an issue with the delete button copy too?

Reply:
And a comma instead of a dash...

![image](https://user-images.githubusercontent.com/68222608/102628761-1e7baa00-4142-11eb-9a52-5a1523a855c4.png)